examples/kprobetcp/README.md

```diff
@@ -27,7 +27,7 @@ cargo build --release
 ## Run
 
 ```bash
-RUST_LOG=info sudo target/release/kprob
+RUST_LOG=info sudo target/release/kprobetcp
```

Comment:
@dmitris Honestly, where did you get that from?

aya-template is using `RUST_LOG=info cargo xtask run` command, which I find more convenient:

https://github.com/aya-rs/aya-template/blob/main/README.md?plain=1#L27

It seems like you didn't generate this project with the newest aya-template, but instead you hand-crafted or copied that README from somewhere.

So to avoid any more surprises - could you please rengenerate this whole project with

```
cargo generate --git https://github.com/aya-rs/aya-template
```

and then just copy/paste your code there?

Reply:
the template-generated code has multiple clippy violations (per `cargo +nightly clippy`) - below is a subset of diffs - so I would be reluctant to fully reuse it "as is".  I copied the README from the template-generated project including `RUST_LOG=info cargo xtask run` run instructions, would it work like this?